bin/circle/search-circle-builds

```diff
@@ -70,15 +84,30 @@ end_build = options[:end_build]
 
 builds = (start_build..end_build).to_a
 
-puts "Checking builds #{builds.join ', '} for errors:\n#{KNOWN_FAIL_TYPES.map { |f| "#{f[:search_string]} in container #{f[:container]} step #{f[:step_string]}" }.join "\n"}"
+fail_types_to_check = KNOWN_FAIL_TYPES
+
+if options[:grep_string]
+  fail_types_to_check = [{
+                             search_string: options[:grep_string],
+                             step_string: options[:grep_step],
+                             container: options[:grep_container]
+                         }]
```

Comment:
Thanks, was trying to find a way to get a sane indentation that Rubocop liked, RubyMine kept auto-formatting to Rubocop-incompatible indentations.